Jesusborjas006/TrackMyBite
# Feat/basic login

### What is the feature?
Create Login page component that asks the user for it's name. This name is then used to render in the home page to greet the user in the nav bar. Styling has been updated along with route navigations. User can logout and insert new name.

### What is the solution?
Created state in the app component to pass user's state in the nav and set user in the login component.

### What areas of the site does it impact?
The new default page is the login input form and styling to the home page has been updated along with a new nav.

### How to test?
Cypress can be used to test if the users name is correctly displayed in the nav bar. Can check if user state is reseted when clicking logout and if user is sent back to the form login page.

## Other Notes
This doesn't keep track of user's meals yet.
